base/broadcast.jl

```diff
@@ -201,22 +201,18 @@ Base.similar(bc::Broadcasted{ArrayConflict}, ::Type{Bool}) =
     similar(BitArray, axes(bc))
 
 ## Computing the result's axes. Most types probably won't need to specialize this.
-broadcast_axes() = ()
-broadcast_axes(A::Tuple) = (OneTo(length(A)),)
-@inline broadcast_axes(A) = axes(A)
 """
     Base.broadcast_axes(A)
```

Comment:
`broadcast_axes` is indeed historical cruft and was a crutch as we transitioned to relying purely on `axes`. I'd even support removing it entirely but it's exported from the `Broadcast` module so that'd be putatively breaking.

Reply:
I was thinking of defining `const broadcast_axes = axes`, if confirmed that it was purely historical.